node/src/main/scala/com/wavesplatform/Importer.scala

```diff
@@ -109,18 +112,18 @@ object Importer extends ScorexLogging {
           override def rollbackTo(blockId: ByteStr): Task[Either[ValidationError, DiscardedBlocks]] =
             Task(blockchainUpdater.removeAfter(blockId)).executeOn(appenderScheduler)
           override def time: Time     = t
-          override def wallet: Wallet = ???
+          override def wallet: Wallet = notImplementedDuringImport
           override def utx: UtxPool   = utxPool
 
-          override def broadcastTransaction(tx: Transaction)                 = ???
-          override def spendableBalanceChanged: Observable[(Address, Asset)] = ???
-          override def actorSystem: ActorSystem                              = ???
-          override def blockchainUpdated: Observable[BlockchainUpdated]      = blockchainUpdatedObservable
-          override def utxEvents: Observable[UtxEvent]                       = Observable.empty
-          override def transactionsApi                                       = ???
-          override def blocksApi                                             = ???
-          override def accountsApi                                           = ???
-          override def assetsApi                                             = ???
+          override def broadcastTransaction(tx: Transaction): TracedResult[ValidationError, Boolean] = notImplementedDuringImport
+          override def spendableBalanceChanged: Observable[(Address, Asset)]                         = notImplementedDuringImport
+          override def actorSystem: ActorSystem                                                      = notImplementedDuringImport
+          override def blockchainUpdated: Observable[BlockchainUpdated]                              = blockchainUpdatedObservable
+          override def utxEvents: Observable[UtxEvent]                                               = Observable.empty
+          override def transactionsApi: CommonTransactionsApi                                        = notImplementedDuringImport
+          override def blocksApi: CommonBlocksApi                                                    = notImplementedDuringImport
```

Comment:
Instead of throwing an exception, please supply a usable instance, just in case the extension needs to actually do something during import.